packages/apps/hr/src/components/atoms/DateDisplay/DateDisplay.tsx

```diff
@@ -35,7 +35,7 @@ const IconDate = ({date, size = 18}: IconDateProps) => {
   return (
     <View style={styles.container}>
       <Icon
-        name="calendar-alt"
+        name="calendar2-week-fill"
```

Comment:
This icon is not adapted to the situation, the component is describingonly one date so maybe more calendar-event